agent.go

```diff
@@ -260,6 +266,25 @@ func (a *Agent) startConnectivityChecks(isControlling bool, remoteUfrag, remoteP
 	a.haveStarted.Store(true)
 	a.log.Debugf("Started agent: isControlling? %t, remoteUfrag: %q, remotePwd: %q", isControlling, remoteUfrag, remotePwd)
 
+	if isControlling {
```

Comment:
I would nominate the first pair that the controlling side sends a `SuccessResponse` for. 

We can come back later and figure out how to pick the 'Highest Priority' later, less chance of bugs for the first round though.